quix-frontend/client/src/lib/runner/services/autocomplete/autocomplete-utils.ts

```diff
@@ -44,6 +45,44 @@ export const createMatchMask = (
   return res;
 };
 
+export async function getSuggestions(queryContext, keywordsCompletions, sqlAutocompleter, searchInObject) {
+  if (queryContext.contextType === ContextType.Undefined) {
+    return keywordsCompletions.filter(obj => obj.value.toLowerCase().includes(queryContext.prefix.toLowerCase()));
+  }
+
+  return searchInObject
+    ? sqlAutocompleter.getCompletionItemsFromQueryContextColumn(
+      queryContext
+    )
+    : sqlAutocompleter.getCompletionItemsFromQueryContext(
+      queryContext
+    );
+
+}
+
+export async function isSearchInObject(queryContext, sqlAutocompleter) {
+  if (queryContext.contextType === ContextType.Column && queryContext.prefix) {
+    const completions = await sqlAutocompleter.getCompletionItemsFromQueryContext(
+      queryContext
+    )
+    const filteredCompletions: object[] = completions.filter(obj => obj.value.toLowerCase().includes(queryContext.prefix));
```

Comment:
obj => obj
better to have one letter like c as we are already close to the "completions" that it the array we itterate over

Reply:
```
const filteredCompletions = completions.filter(c => c.value.toLowerCase().includes(queryContext.prefix));